test/system/104-run.bats

```diff
@@ -243,22 +243,22 @@ teardown() {
   assert_output "other-container"
 }
 
-@test "run: Ensure that $HOME is used as a fallback working directory" {
-  local default_container_name="$(get_system_id)-toolbox-$(get_system_version)"
-  create_default_container
-
-  pushd /etc/kernel
-  run --separate-stderr $TOOLBOX run pwd
-  popd
-
-  assert_success
-  assert_line --index 0 "$HOME"
-  assert [ ${#lines[@]} -eq 1 ]
-  lines=("${stderr_lines[@]}")
-  assert_line --index $((${#stderr_lines[@]}-2)) "Error: directory /etc/kernel not found in container $default_container_name"
-  assert_line --index $((${#stderr_lines[@]}-1)) "Using $HOME instead."
-  assert [ ${#stderr_lines[@]} -gt 2 ]
-}
+#@test "run: Ensure that $HOME is used as a fallback working directory" {
+#  local default_container_name="$(get_system_id)-toolbox-$(get_system_version)"
```

Comment:
I see that you have commented out this test, but in the test logs it still seems to be running:
```
fedora-38 | not ok 127 run: Ensure that /home/zuul-worker is used as a fallback working directory in 3054ms
fedora-38 | # (in test file test/system/104-run.bats, line 251)
fedora-38 | #   `run -127 --separate-stderr $TOOLBOX run pwd' failed, expected exit code 127, got 0
fedora-38 | # /etc/kernel ~/src/github.com/containers/toolbox
fedora-38 | # /home/zuul-worker
fedora-38 | # stderr:
fedora-38 | # Error: crun: chdir `/etc/kernel`: No such file or directory: OCI runtime attempted to invoke a command that was not found
fedora-38 | # Error: directory /etc/kernel not found in container fedora-toolbox-38
fedora-38 | # Using /home/zuul-worker instead.
```

I am not sure what's going on.  :)

Reply:
Okay, it dropped out of the test logs after the last `recheck`.  Now let's see why it was failing.

Reply:
I took the liberty to `git push --force` to your branch to restore the tests and the logs say that it's passing:
```
fedora-38 | ok 127 run: Ensure that /home/zuul-worker is used as a fallback working directory in 3208ms
```

Are you sure you didn't have any other local changes?

If you want, you can re-create your local branch with:
```
$ git checkout main
$ git branch -D dbus-test
$ git fetch -p nievesmontero
$ git branch --track dbus-test nievesmontero/dbus-test
```

Reply:
I've followed the steps and re-created my local branch.